iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -80,6 +82,26 @@ public TypeDeclaration TYPE_DECLARATION() {
         STRING_LITERAL()));
   }
 
+  // TODO SONARIAC-967 Put in place decorator
+  public OutputDeclaration OUTPUT_DECLARATION() {
+    return b.<OutputDeclaration>nonterminal(BicepLexicalGrammar.OUTPUT_DECLARATION).is(
+      b.firstOf(
+        f.outputDeclaration(
+          b.token(BicepKeyword.OUTPUT),
+          IDENTIFIER(),
+          IDENTIFIER(),
+          b.token(Punctuator.EQU),
+          EXPRESSION()),
+        f.outputDeclaration(
+          b.token(BicepKeyword.OUTPUT),
+          IDENTIFIER(),
+          b.token(BicepKeyword.RESOURCE),
+          // TODO SONARIAC-958 Put in place interpString
+          STRING_LITERAL(),
```

Comment:
I think you can already use the rule which is called `INTERPOLATED_STRING_TYPE` in the current master, which is simply a quoted string literal.